src/app.js

```diff
@@ -1 +1,23 @@
+/* eslint-disable no-console */
 'use strict';
+
+function operate(oper, nums) {
+  if (oper !== 'add' && oper !== 'multiply') {
```

Comment:
better use switch case after length check.
Check the action and return error by default.